rfhub2/cli/rfhub_importer.py

```diff
@@ -56,7 +71,17 @@ def import_libraries(self) -> Tuple[int, int]:
         """
         libraries_paths = self.get_libraries_paths()
         collections = self.create_collections(libraries_paths)
-        loaded_collections = self.add_collections(collections)
+        if self.mode == "append":
+            loaded_collections = self.add_collections(collections)
+        elif self.mode == "insert":
+            self.delete_all_collections()
+            loaded_collections = self.add_collections(collections)
+        else:
+            existing_collections = self.get_all_collections()
+            self.delete_outdated_collections(existing_collections, collections)
```

Comment:
> Maybe delete step could be put after the update step?

Good point. Better to have collections twice than not at all.



> I think that in the future it would be best to move the update logic to the server endpoint to make this operation safer for data integrity.

You mean like to have temporary tables and than swap or something totally different?

Reply:
I swapped order of commands